### PR TITLE
DOP-3571: Add README.md

### PR DESCRIPTION
**Pull request by casthewiz**
Adds a barebones README.md file to the repository. 
Additional testing details to come as we setup CI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+A node debugger (ie. [chrome developer tools](https://nodejs.org/en/docs/guides/debugging-getting-started/#inspector-clients)) can be connected to the built JS files.
+
+## Staging
+All commits merged to `main` deploy via Kanopy to a staging instance. 
+The staging instance is viewable at https://docs-search-transport.docs.staging.corp.mongodb.com/status. 
+
+## Releasing
+New release tags automatically begain deployment via Kanopy to production instances.
```
Comment (seungpark) [inline README.md:23]: minor typo. "begin"

**Review by seungpark — CHANGES_REQUESTED** [review]
LGTM! very informative. can we add an overview of this module below the first heading?

ie `REST server that manages search documents within an Atlas cluster and handles user search requests, deployed via Kanopy drones`

**Comment by casthewiz** [comment]
Haha, that would indeed be good to add!

**Review by seungpark — APPROVED** [review]
LGTM! much thanks for adding dotenv! 🙏

**Pull request merged**